CHANGELOG_CLOUD.md

```diff
@@ -0,0 +1,682 @@
+# Changelog
+All notable changes to this project for *Dgraph's cloud releases* will be documented in this file.
```

Comment:
nit: may be change `Dgraph's cloud releases ` to more meaningful for a external oss user.